src/fragments/lib/logging/ios/escapehatch/escapehatch.mdx

```diff
@@ -0,0 +1,13 @@
+Add import statements
+
+```swift
```

Comment:
can we have these sections of code snippets be in the same code snippet?

Reply:
as previously commented, import statements are separate to avoid developer confusion when copying/pasting code.  I can combine it if you still think it should be combined.